package.json

```diff
@@ -101,9 +101,9 @@
     "release": "npm build && electron-builder --publish=always",
     "build": "npm build && npm electron-build",
     "start": "concurrently \"cross-env BROWSER=none npm run b-start\" \"node scripts/await.js && electron .\"",
-    "lint": "eslint src/**/*.ts* cypress/**/*.ts*",
-    "lint-fix": "eslint src/**/*.ts* cypress/**/*.ts* --fix",
-    "lint-warn": "eslint src/**/*.ts* cypress/**/*.ts* --max-warnings=0",
+    "lint": "eslint .",
+    "lint-fix": "eslint . -c .eslintrc.pre-push.json --fix",
+    "lint-warn": "eslint . -c .eslintrc.pre-push.json --max-warnings=0",
```

Comment:
Nie wiem szczerze mówiąc dlaczego, ale z poprzednim regexem linter nie przechodził przez wszystkie pliki.
Zamieniłem na kropkę i jeśli nie było jakichś specjalnych powodów dlaczego mielibyśmy używać wcześniejszego regex'a, to proponuję przy niej zostać (bo jest krótsza)

Reply:
Oki, spoko